pkg/platform/provider/baremetal/cluster/kubeadm.go

```diff
@@ -230,9 +228,6 @@ func (p *Provider) getSchedulerExtraArgs(c *v1.Cluster) map[string]string {
 		"use-legacy-policy-config": "true",
 		"policy-config-file":       constants.KuberentesSchedulerPolicyConfigFile,
 	}
-	for k, v := range c.Spec.SchedulerExtraArgs {
-		args[k] = v
-	}
 
```

Comment:
why remove above lines?

Reply:
it's redundant. the assignment has been done in the next line.